doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1093,6 +1093,8 @@ The Dataverse Software S3 driver supports multi-part upload for large files (ove
 First: Set Up Accounts and Access Credentials
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+**Note:** As of version 5.14, if Dataverse is running in an EC2 instance it will prefer RBAC for S3, even if administrators configure Dataverse with programmatic access keys. This is preferential from a security perspective as there are no keys to rotate or have stolen. If you intend to assign a role to your EC2 instance, you will still need the ``~/.aws/config`` file to specify the region but you need not generate credentials. For more information please see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2.html
+
```

Comment:
FWIW: From https://github.com/IQSS/dataverse/blob/637c21684d45d8d176cd13bec1481877311959df/src/main/java/edu/harvard/iq/dataverse/dataaccess/S3AccessIO.java#L1203-L1210 I think you can still use a profile but RBAC wins over the default profile.

Reply:
yea i believe that was our experience as we've been running with a profile for months, years, but SSM somehow added the instance profile with it's default role and that took priority

Reply:
@qqmyers an excellent catch, and thank you for pointing this out!